package.json

```diff
@@ -88,7 +88,7 @@
     "@storybook/addon-links": "^7.0.0-beta.47",
     "@storybook/react": "^7.0.0-beta.47",
     "@storybook/react-webpack5": "^7.0.0-beta.47",
-    "@storybook/testing-library": "^0.0.13",
+    "@storybook/testing-library": "^0.0.14-next.1",
```

Comment:
I think this is fine but just curious, what comes with this upgrade?

Reply:
We got an error and the solution from: https://github.com/storybookjs/testing-library/issues/32

> It looks like this is because you are using storybook v7. The release notes for @storybook/testing-library show that support for storybook v7 is being added in @storybook/testing-library v0.0.14.
To get it working right now you can install the next release, currently it is @storybook/testing-library@v0.0.14-next.1.